src/psyclone/dynamo0p3.py

```diff
@@ -106,8 +107,9 @@
 
 # The mapping from meta-data strings to field-access types
 # used in this API.
+# Note that readwrite is not currently supported.
 FIELD_ACCESS_MAP = {"write": "gh_write", "read": "gh_read",
-                    "inc": "gh_inc"}
+                    "inc": "gh_inc", "readwrite": ""}
```

Comment:
I am currently adding support for "readwrite" (issue #25) so I am introducing the uses based on @arporter's related modifications and comments.

Reply:
A couple of the methods in `psyGen.Kern` expect the access map dictionary to contain the "readwrite" key. If it isn't present then we get exceptions. I've therefore added an empty entry here to avoid errors. I've extended the comment slightly.